src/client/utils/getCookie.js

```diff
@@ -0,0 +1,5 @@
+export function getCookie (name) {
+	const regex = new RegExp(`${name}=([^;]+)/`);
```

Comment:
There is a trailing `/` which I think was from when a literal regex was used.

```suggestion
	const regex = new RegExp(`${name}=([^;]+)`);
```

This regex would match on the suffix of a cookie name, should we update the regex to only match a full cookie name?

If so, then this would work:
```suggestion
	const regex = new RegExp(`(?:^| )${name}=([^;]+)`);
```

Reply:
good spot, and that's a good idea, thanks!